src/wallet/wallet.h

```diff
@@ -1345,6 +1346,16 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
 
     //! Add a descriptor to the wallet, return a ScriptPubKeyMan & associated output type
     ScriptPubKeyMan* AddWalletDescriptor(WalletDescriptor& desc, const FlatSigningProvider& signing_provider, const std::string& label, bool internal);
+
+    std::map<uint256, CQueuedTransaction> queuedTransactionMap;
+
+    bool QueuedTransactionExists(const uint256 &txid);
+    bool WriteQueuedTransaction(
+            const uint256 &txid,
+            const CMutableTransaction &tx);
+    bool EraseQueuedTransaction(const uint256 &txid);
+    bool GetQueuedTransaction(const uint256 &txid, CQueuedTransaction *data=nullptr);
+    void EffectTransactionQueue();
```

Comment:
Could `EffectTransactionQueue` be `private`?  I believe it is only called from inside the wallet itself, right?

Reply:
Yes.